client/src/styles/layout.css

```diff
@@ -4,103 +4,104 @@ Permission to use, copy, modify, and distribute this software and its documentat
 
 The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
 */
-html, body {
-    height: 100%;
-    width: 100%;
-    text-align: center;
-    margin: 0;
+html,
+body {
+  height: 100%;
```

Comment:
it looks like your IDE might have changed the css format and maybe prettier isn't controlling css format? We definitely don't want format-only changes

Reply:
I already added `css` to package.json and fixed this issue